src/fpm_command_line.f90

```diff
@@ -245,9 +248,11 @@ subroutine get_command_line_settings(cmd_settings)
           ' --no-prune F' // &
           ' --compiler "'//get_fpm_env(fc_env, fc_default)//'"' // &
           ' --c-compiler "'//get_fpm_env(cc_env, cc_default)//'"' // &
+          ' --cpp-compiler "'//get_fpm_env(cxx_env, cxx_default)//'"' // &
```

Comment:
This is also inconsistent, given the environment variable choice we should use `cxx` consistently for all C++ related variables